pkgs/misc/cups/drivers/estudio/default.nix

```diff
@@ -0,0 +1,78 @@
+{ stdenv, fetchurl, perl }:
+
+# Driver suitable for the following printers:
+# TOSHIBA e-STUDIO2000AC
+# TOSHIBA e-STUDIO2005AC
+# TOSHIBA e-STUDIO2040C
+# TOSHIBA e-STUDIO2050C
+# TOSHIBA e-STUDIO2055C
+# TOSHIBA e-STUDIO2500AC
+# TOSHIBA e-STUDIO2505AC
+# TOSHIBA e-STUDIO2540C
+# TOSHIBA e-STUDIO2550C
+# TOSHIBA e-STUDIO2555C
+# TOSHIBA e-STUDIO287CS
+# TOSHIBA e-STUDIO3005AC
+# TOSHIBA e-STUDIO3040C
+# TOSHIBA e-STUDIO3055C
+# TOSHIBA e-STUDIO347CS
+# TOSHIBA e-STUDIO3505AC
+# TOSHIBA e-STUDIO3540C
+# TOSHIBA e-STUDIO3555C
+# TOSHIBA e-STUDIO407CS
+# TOSHIBA e-STUDIO4505AC
+# TOSHIBA e-STUDIO4540C
+# TOSHIBA e-STUDIO4555C
+# TOSHIBA e-STUDIO5005AC
+# TOSHIBA e-STUDIO5055C
+# TOSHIBA e-STUDIO5506AC
+# TOSHIBA e-STUDIO5540C
+# TOSHIBA e-STUDIO5560C
+# TOSHIBA e-STUDIO6506AC
+# TOSHIBA e-STUDIO6540C
+# TOSHIBA e-STUDIO6550C
+# TOSHIBA e-STUDIO6560C
+# TOSHIBA e-STUDIO6570C
+# TOSHIBA e-STUDIO7506AC
+
+stdenv.mkDerivation {
+  name = "cups-toshiba-estudio";
+  version = "7.51";
+
+  src = fetchurl {
+    url = http://business.toshiba.com/downloads/KB/f1Ulds/14079/TOSHIBA_ColorMFP_CUPS.tar;
+    sha256 = "3741bb79723495da5cb5a3971ae8c6042b6c71a6264af8f25aecf721f1f0752f";
+  };
+
+  buildInputs = [ perl ];
+
+  phases = [ "unpackPhase"
+             "preInstall"
+             "installPhase" ];
+
+  preInstall = ''
```

Comment:
can you call this the configurePhase or patchPhase? pre-phases are supposed to be called from the phase itself